app/src/main/java/ch/epfl/sdp/peakar/camera/CameraActivity.java

```diff
@@ -137,14 +143,15 @@ private CompassListenerInterface getCompassListener() {
     }
 
     /**
-     * onPause release the sensor listener from compass when user leave the application
+     * onPause release the sensor listener from compass when user leaves the application
      * without closing it (app running in background)
      */
     @Override
     protected void onPause() {
         super.onPause();
         // releases the sensor listeners of the compass
         compass.stop();
+        //addDiscoveredPOIsToDatabase();
```

Comment:
Will this be uncommented later?

Reply:
This was a leftover comment and is now removed

Reply:
It was a leftover and is removed now.